app/src/controllers/roadmap.ts

```diff
@@ -47,6 +50,29 @@ const controller = {
 
       // Send the email
       const { data, status } = await emailService.email(req.body.emailData);
+
+      // Add a new note on success
+      if (status === 201) {
+        const userId = await userService.getCurrentUserId(getCurrentIdentity(req.currentUser, NIL), NIL);
+
+        let noteBody = req.body.emailData.body;
```

Comment:
You could put the roadmap subject in the note as well. and who it was sent to.. but i see Tyler didnt mention that so guess leave it like this.